src/rgw/rgw_lc.h

```diff
@@ -91,6 +91,43 @@ class LCExpiration
 };
 WRITE_CLASS_ENCODER(LCExpiration)
 
+class LCFilter
+{
+ protected:
+  std::string prefix;
+  // TODO add support for tagging
+ public:
+  std::string& get_prefix(){
```

Comment:
(I guess come to think of it, is there a real point to having an accessor or mutator here at all? If this isn't being used through some interface somewhere, maybe it's more straightforward without them. Seems ok as-is, but something to consider.)